analyses/WA_cd_2010/doc_WA_cd_2010.md

```diff
@@ -0,0 +1,24 @@
+# 2010 Washington Congressional Districts
+
+## Redistricting requirements
+In Washington, districts must according to HCR 4409 (https://apps.leg.wa.gov/billsummary/?BillNumber=4409&Year=2011&Initiative=false):
+
+1. be contiguous
+2. have equal populations
+3. be geographically compact
+4. preserve county and municipality boundaries as much as possible
+
+
+### Interpretation of requirements
+We enforce a maximum population deviation of 0.5%. We also used ferry routes in order to create districts linking precincts that are offshore. 
```

Comment:
we do more than just ferries: we do highways etc. too.  Can you look at the documentation for WA 2020 and make this file look like that? Not just in this section but throughout.

https://github.com/alarm-redist/fifty-states/pull/99